crates/aranya-daemon-api/src/service.rs

```diff
@@ -95,35 +102,38 @@ custom_id! {
 }
 
 custom_id! {
-    /// An AQC label ID.
-    pub struct LabelId;
-}
-
-custom_id! {
-    /// An AQC bidi channel ID.
-    pub struct AqcBidiChannelId;
+    /// A role ID.
+    pub struct RoleId;
 }
 
-custom_id! {
-    /// An AQC uni channel ID.
-    pub struct AqcUniChannelId;
+#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+pub struct Role {
+    /// Uniquely identifies the role.
+    pub id: RoleId,
+    /// The role's friendly name.
+    pub name: Text,
+    /// The author of the role.
+    pub author_id: DeviceId,
+    /// Is this a default role?
+    pub default: bool,
 }
 
 /// A device's public key bundle.
-#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+#[derive(Clone, Serialize, Deserialize, Eq, PartialEq)]
 pub struct KeyBundle {
     pub identity: Vec<u8>,
     pub signing: Vec<u8>,
     pub encoding: Vec<u8>,
 }
 
-/// A device's role on the team.
-#[derive(Copy, Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
-pub enum Role {
-    Owner,
-    Admin,
-    Operator,
-    Member,
+impl fmt::Debug for KeyBundle {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("KeyBundle")
+            .field("identity", &Hex::new(&*self.identity))
+            .field("signing", &Hex::new(&*self.signing))
+            .field("encoding", &Hex::new(&*self.encoding))
```

Comment:
We've fixed this upstream:
```suggestion
            .field("encryption", &Hex::new(&*self.encryption))
```